src/OpenTelemetry/CHANGELOG.md

```diff
@@ -44,6 +44,15 @@
   before setting up the `MeterProvider`.
   ([#5052](https://github.com/open-telemetry/opentelemetry-dotnet/pull/5052))
 
+* Update Metrics SDK to override the default histogram buckets for ASP.NET
+  (.NET Framework) and the specific instrument:
+  * `http.request.server.duration`
+
+  Histogram metrics which have their `Unit` as `s` (second) will have their
```

Comment:
```suggestion
  Histogram metrics with meter name abc, instrument name xyz, `Unit` as `s` (second) will have their
```

Reply:
The instrument is already mentioned, and none of the other changelogs ever mentions the meter names. Maybe it is fine to leave as-is?

Reply:
Having the meter name will make it more clear which specific case(`OpenTelemetry.Instrumentation.AspNet`) we are referring to here. 

Other changelogs do mention runtime(agree it could be improved further).

Reply:
I did some changes to the wording, let me know if I need to make any other changes.